test/data/ej-fra-package.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version='1.0' encoding='utf-8'?>
+<TEI>
+    <!-- To exclude any element from alignment, add the do-not-align="true" attribute to
+         it, e.g., <p do-not-align="true">...</p>, or
+         <s>Some text <foo do-not-align="true">do not align this</foo> more text</s> -->
+    <text xml:lang="fra">
+        <body>
+            <div type="page">
+                <graphic url="data/diamond.png"/>
+                <p>
+                    <s>Bonjour.</s>
+                    <s>Je m'appelle Éric Joanis.</s>
+                    <s>Je suis programmeur au sein de l'équipe des technologies pour les langues autochtones au CNRC.</s>
+                </p>
+            </div>
+            <div type="page">
+                <graphic url="https://cdn.shibe.online/shibes/bdf0a814c42e39e42870ffb22075ee0a9f1db933.jpg"/>
```

Comment:
I'm still not comfortable with the use of this image in our test suite. Free sites are often only free for personal use, but require a paid license for use by governments and corporations. Can we use something we own, as in the NRC? How about https://nrc.canada.ca/libraries/theme-gcweb/assets/sig-blk-en.svg ?

Reply:
Of course, the link might break in the future, so it's maybe not ideal, but if we use something from shibe, I'd like to keep a link to the terms of use right here to prove we're allowed to use it.